app/scripts/ui.js

```diff
@@ -362,29 +290,30 @@ async function displayCriticalError(errorKey, err, metamaskState) {
 }
 
 /**
- * Establishes a connection to the background and a Web3 provider
+ * Establishes a connections between the PortStream (background) and various UI
+ * streams.
  *
- * @param {PortDuplexStream} connectionStream - PortStream instance establishing a background connection
+ * @param {PortStream} connectionStream - PortStream instance establishing a background connection
+ * @returns The multiplexed streams
  */
-function connectToAccountManager(connectionStream) {
+function connectSubstreams(connectionStream) {
```

Comment:
renamed because `connectToAccountManager` is uh, maybe what it does, but it just does _so much more_ than that.